generator/nidaqmx_python_generator/metadata/attributes.py

```diff
@@ -0,0 +1,130 @@
+                    "cviName": "channel"
+                },
+            },
+            'python_class_name':'AI_Channel'
+        },
+        12176: {
+            'access': 'read-write',
+            'name': 'AI_BRIDGE_POLY_FORWARD_COEFF',
+            'resettable': True,
+            'type': 'float64[]',
+            'ctypes_data_type': 'numpy.float64',
+            'python_data_type': 'float',
+            'description': 'Specifies an list of coefficients for the polynomial that converts electrical values to physical values. Each element of the list corresponds to a term othe equation. For example, if index three of the list is 9,the fourth term of the equation is 9x^3.',
+            'has_explicit_read_buffer_size': True,
+            'bitfield_enum':'N/A',
+            'is_list':True,
+            'calling_convention':'StdCall',
+            'c_function_name':'AIBridgePolyForwardCoeff',
+            'handle_parameters':
+            {
+                'taskHandle':{
+                    "accessor": "self._handle", 
+                    "ctypes_data_type": "lib_importer.task_handle", 
+                    "cviName": "taskHandle"
+                },
+                'ChannelHandle':{
+                    "accessor": "self._name", 
+                    "ctypes_data_type": "ctypes.c_char_p", 
+                    "cviName": "channel"
+                }
+            },
+            'is_object': False,
+            'python_class_name':'AI_Channel'
+        },
+        4148: {
+            'access': 'read',
+            'name': 'AI_THRMCPL_CJC_CHAN',
+            'resettable': False,
+            'type': 'char[]',
+            'ctypes_data_type': 'ctypes.c_char_p',
+            'python_data_type': 'str',
+            'description': 'Indicates the channel that acquires the temperature of the cold junction if **ai_thrmcpl_cjc_src** is **CJCSource1.SCANNABLE_CHANNEL**. If the channel is a temperature channel, NI-DAQmx acquires the temperature in the correct units. Other channel types, such as a resistance channel with a custom sensor, must use a custom scale to scale values to degrees Celsius.',
+            'has_explicit_read_buffer_size': True,
+            'bitfield_enum':'N/A',
+            'is_list':False,
+            'calling_convention':'StdCall',
+            'c_function_name':'AIThrmcplCJCChan',
+            'handle_parameters':
+            {
+                'taskHandle':{
+                    "accessor": "self._handle", 
+                    "ctypes_data_type": "lib_importer.task_handle", 
+                    "cviName": "taskHandle"
+                },
+                'ChannelHandle':{
+                    "accessor": "self._name", 
+                    "ctypes_data_type": "ctypes.c_char_p", 
+                    "cviName": "channel"
+                }
+            },
+            'is_object': True,
+            'object_type': 'Channel',
+            'object_has_factory': True,
+            'objectConstructorParams':
```

Comment:
`objectConstructorParams` is still camelCase.

Reply:
@bkeryan I saw some of the other multi worded metadata in functions.py for daqmx which were of snake case (these were grpc related metadata), hence i moved all data to snake case

Reply:
ok, I see: https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nidaqmx/functions.py#L5625

```
    'CreateTask': {
        'init_method': True,
        'parameters': [
            {
                'direction': 'in',
                'is_session_name': True,
                'name': 'sessionName',
                'type': 'const char[]'
            },
            {
                'direction': 'out',
                'name': 'task',
                'type': 'TaskHandle'
            },
            {
                'cppName': 'initializationBehavior',
                'direction': 'in',
                'grpc_type': 'nidevice_grpc.SessionInitializationBehavior',
                'name': 'initializationBehavior',
                'proto_only': True,
                'type': 'int32'
            },
            {
                'cppName': 'newSessionInitialized',
                'direction': 'out',
                'grpc_type': 'bool',
                'name': 'newSessionInitialized',
                'proto_only': True,
                'type': 'bool'
            }
        ],
        'returns': 'int32'
    },
```

Reply:
So based on this we could stick with snake case right?